packages/shared/src/components/squads/SquadPageHeader.tsx

```diff
@@ -95,16 +98,28 @@ export function SquadPageHeader({
       )}
       <div
         className={classNames(
-          'absolute bottom-0 w-full translate-y-1/2 px-6 laptopL:px-0 bg-theme-bg-primary',
+          'relative tablet:absolute flex flex-col tablet:flex-row justify-center items-center pt-8 tablet:p-0 bottom-0 w-full tablet:translate-y-1/2 laptopL:px-0 bg-theme-bg-primary laptop:max-w-[41.5rem]',
           shouldShowHighlightPulse && 'highlight-pulse',
-          MAX_WIDTH,
         )}
       >
+        <Divider />
         <SharePostBar
-          className="w-full"
+          className="w-full max-w-[30.25rem]"
```

Comment:
Can we standardize this size maybe to some rounded value? Non blocking.

Reply:
I think this should be acceptable as the equivalent would still be divisible by 4. Though we might want to store these values somewhere so we can standardize the usage. WDYT?z

Reply:
I was thinking of introducing more sizes in our design system like `w-121` class would be equivalent, but not sure if can be standardized inside design.

Reply:
Ah, right. That's actually better, I see these values also being used more often now. Let's probably discuss this in the web team meeting and coordinate with the design team what should we label these sizes.

Reply:
🤝 agreed